.travis.yml

```diff
@@ -11,4 +11,4 @@ install:
 
 script:
     - flake8 . --show-source --statistics
-    - docker run baselines-test pytest -v .
```

Comment:
is forked better than using xdist?

Reply:
No, but I think travis do not have enough cores to actually use parallel version of pytest; here forked is needed rather for test isolation (the new video recorder tests freeze without it)

Reply:
Ah, my only concern is that pytest-forked has a warning about maintenance: https://github.com/pytest-dev/pytest-forked#pytest-forked-run-each-test-in-a-forked-subprocess while xdist has no such warning.